scripts/update_pytorch_pin.sh

```diff
@@ -0,0 +1,27 @@
+#! /bin/bash
```

Comment:
We already have a script for updating the pins for XLA, JAX, etc. Can we merge this into that script?